Applications/Custom/LayerClient/res/custom_layer_client.ini

```diff
@@ -26,3 +26,9 @@ Type = fully_connected
 unit = 10
 Bias_initializer = zeros
 Activation = softmax
+
+# Below is the custom loss layer,
+# This will directly replace loss in the Model section.
+# Thus, there should be no loss provided in the model section.
+[losslayer]
+Type = mae_loss
```

Comment:
I'm sure it it. But just check if there is type check between custom::mae_loss and mae_loss.

Reply:
Yes there is, if mae_loss exist, custom::mae_loss will fail to register. so the developer of mae_loss must have to choose different type name 👍